easybuild/easyblocks/t/tensorflow.py

```diff
@@ -281,11 +283,6 @@ def build_step(self):
         if cuda_root:
             cmd.append('--config=cuda')
 
-        # enable mkl-dnn by default, but only if cuDNN is not listed as dependency
-        if self.cfg['with_mkl_dnn'] is None and get_software_root('cuDNN') is None:
-            self.log.info("Enabling use of mkl-dnn since cuDNN is not listed as dependency")
-            self.cfg['with_mkl_dnn'] = True
```

Comment:
@akesandgren Can you clarify this? Can both be used together? There must have been a reason why did this?

Reply:
We did this because i originally thought there would be a problem, but I later tried and they can be compiled in at the same time.
So it's basically removing a bit of code that shouldn't have been there in the first place.